public/img/blue-checkmark.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we get rid of the black checkmark and circle files? Or are they still being used elsewhere?

Reply:
they're being used when the inputs don't have focus

Reply:
Ah, right. Nevermind!